infrastructure/main.tf

```diff
@@ -75,11 +75,18 @@ module "recipe-database" {
   sku_name           = "GP_Gen5_2"
   sku_tier           = "GeneralPurpose"
   storage_mb         = "51200"
-  common_tags        = var.common_tags
```

Comment:
this is built via jenkins, any missing tags should be added to cnp-jenkins-library